### PR TITLE
Infinite scroll sample

### PR DESCRIPTION
**Pull request by MayaKirova**
There are some issues that are worked around in the sample:
1) Seems that DataPreLoad does not fire if the loaded data is the same number as the `totalItemCount` so there's a workaround that sets it to number of records +1 .
2) There is one additional wait workaround due to https://github.com/IgniteUI/igniteui-angular/issues/13242

### CODE REVIEW

**Comment by HUSSAR-mtrela** [comment]
waiting for @dkamburov review

File: samples/grids/grid/infinite-scroll/App.razor [hunk 1]
```diff
@@ -0,0 +1,131 @@
+@using IgniteUI.Blazor.Controls
+@using System.Net.Http;
+@using Newtonsoft.Json;
+
+<IgbGrid AutoGenerate="false"
```
Comment (HUSSAR-mtrela) [inline samples/grids/grid/infinite-scroll/App.razor:5]: the grid should be wrapped in DIV, e.g.
<div class="container vertical">

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/grids/grid/infinite-scroll/App.razor (hunk 1 above)
Comment (MayaKirova) [reply samples/grids/grid/infinite-scroll/App.razor:5]: @HUSSAR-mtrela  Ok. But what is the reason it needs to be wrapped in a div? I haven't noticed any issues when there's no div.

Comment (HUSSAR-mtrela) [reply samples/grids/grid/infinite-scroll/App.razor]: to ensure that all samples have the same styling on their container and CSS styling from docs does not show up in our samples

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**